openfl/experimental/utilities/metaflow_utils.py

```diff
@@ -301,18 +316,20 @@ def create_run(self):
 
     def create_task(self, task_name):
         # May need a lock here
-        if self.backend == 'ray':
-            task_id = ray.get(self.counter.get_counter.remote())
+        if self.backend == "ray":
+            with SystemMutex("critical_setcion"):
```

Comment:
Spelling: "critical_section"

Reply:
Fixed the spelling mistake and can observe the change in the required commit : b3b7294